shaders/src/light_reflections.fs

```diff
@@ -195,13 +195,12 @@ highp mat4 scaleMatrix(const highp float x, const highp float y) {
  * r is the desired reflected vector.
  *
  * Fr.a is set to a value between [0, 1] representing the "opacity" of the reflection. 1.0f is full
- * screen-space reflection. Values < 1.0f should be blended with the scene's IBL. Fr.rgb is
- * premultiplied by Fr.a.
+ * screen-space reflection. Values < 1.0f should be blended with the scene's IBL.
```

Comment:
Remove the word "premultiplied" from line 194

Reply:
ah yeah, sorry about that btw. When working on the code I realized that it was more logical for it to not be premultiplied. Similarly to the refraction.